hr_employee_phone_extension/__init__.py

```diff
@@ -0,0 +1,4 @@
+# copyright 2013 Savoir-faire Linux (<http://www.savoirfairelinux.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
```

Comment:
@nikul-serpentcs ,
I think we need to remove the license from `__init__.py` file right?